.github/workflows/tag.yml

```diff
@@ -0,0 +1,36 @@
+name: Tag a release
+on:
+  workflow_dispatch:
+    inputs:
+      branchToTag:
+        description: 'Branch to tag'
+        required: true
+      newTag:
+        description: 'New tag (ex: v1.0.0-alpha1)'
+        required: true
+jobs:
+  tagRelease:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: refs/heads/${{ github.event.inputs.branchToTag }}
+
+    - name: NPM - Test, Build, Deploy
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.11.1
+        registry-url: https://registry.npmjs.org/
+
+    - name: Cache Yarn Deps
+      uses: c-hive/gha-yarn-cache@v1
+
+    - run: yarn install --frozen-lockfile
+    - run: yarn lerna version --no-push --no-commit-hooks --force-publish --no-git-tag-version --yes ${{ github.event.inputs.newTag }}
```

Comment:
This is missing the `--sign-git-commit --sign-git-tag` options from `release:tag`, just making sure that's expected. Also, with this action added, should we remove the `release:tag` command? Or any other publish-related commands?

Reply:
We'd have to provide a secret GPG key to sign (actions can't yet sign using the keys provided by users to sign commits via the github GUI)  RE: https://github.com/actions/runner/issues/667

For the sake of progress, I think I'm willing to give up signed tags 😢